docs/gl_objects/keys.rst

```diff
@@ -0,0 +1,28 @@
+####
+Keys
+####
+
+Keys
+====
+
+Reference
+---------
+
+* v4 API
+
+  + :class:`gitlab.v4.objects.Key`
+  + :class:`gitlab.v4.objects.KeyManager`
+  + :attr:`gitlab.Gitlab.keys`
+
+* GitLab API: https://docs.gitlab.com/ce/api/keys.html
```

Comment:
Can we have just a quick note that for SSH keys, this requires admin access? Like here:
https://github.com/python-gitlab/python-gitlab/blob/eabe091945d3fe50472059431e599117165a815a/docs/gl_objects/variables.rst#L12

Reply:
And done.